include/Lv2Ports.h

```diff
@@ -143,31 +144,33 @@ struct ControlPortBase : public PortBase
 	std::vector<float> m_scalePointMap;
 };
 
-struct Control : public ControlPortBase
+struct Control : public VisitablePort<Control, ControlPortBase>
 {
-	IS_PORT_TYPE
-
 	//! Data location which Lv2 plugins see
 	//! Model values are being copied here every run
 	//! Between runs, this data is not up-to-date
 	float m_val;
+
+	// overwrite accept() from ControlPortBase
+	void accept(Visitor& v) override { v.visit(*this); }
+	void accept(ConstVisitor& v) const override { v.visit(*this); }
```

Comment:
These manual overrides shouldn't be necessary now the `VisitablePort` struct template is used. (Same below in `Cv`.)

Reply:
Oh, indeed. That's what we actually introduced `VisitablePort` for. Fixed.